site/src/lib/blocks.ts

```diff
@@ -1,8 +1,9 @@
 import { BlockMetadata, BlockMetadataRepository } from "blockprotocol";
-// eslint-disable-next-line no-restricted-imports,unicorn/prefer-node-protocol -- https://github.com/vercel/nft/issues/293
-import fs from "fs";
-import glob from "glob";
+import fs from "fs-extra";
+import { globby } from "globby";
 import hostedGitInfo from "hosted-git-info";
+// eslint-disable-next-line unicorn/prefer-node-protocol -- Replace "path" with "node:path" after upgrading next from 12.1.6 https://github.com/vercel/nft/issues/293
+import path from "path";
```

Comment:
A newer version of Next should include an updated version of `@vercel/nft`, which should allow us to restore `node:` prefix. I can keep this PR open till then. Context: https://github.com/vercel/nft/pull/285

Reply:
This problem is no longer present in Next `12.1.7-canary.44`. I suggest waiting for a stable release of Next.js before finishing this PR.